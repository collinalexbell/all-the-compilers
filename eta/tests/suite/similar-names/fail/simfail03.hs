{-# LANGUAGE TypeFamilies #-}

data family A a

data instance A Int    = Baa Int
data instance A Char   = BAa Char
data instance A Double = BAA Double

data AB = BAA

main = return ()
