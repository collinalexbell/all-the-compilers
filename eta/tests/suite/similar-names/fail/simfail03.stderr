
tests/suite/similar-names/fail/simfail03.hs:6:26: error:
    Multiple declarations with names differing only in case.
    Declared at: tests/suite/similar-names/fail/simfail03.hs:5:26
                 tests/suite/similar-names/fail/simfail03.hs:6:26
                 tests/suite/similar-names/fail/simfail03.hs:7:26
                 tests/suite/similar-names/fail/simfail03.hs:9:11

tests/suite/similar-names/fail/simfail03.hs:9:11: error:
    Multiple declarations of â€˜BAAâ€™
    Declared at: tests/suite/similar-names/fail/simfail03.hs:7:26
                 tests/suite/similar-names/fail/simfail03.hs:9:11
